katas/content/superdense_coding/bob_decodes_message/Placeholder.qs

```diff
@@ -0,0 +1,14 @@
+namespace Kata {
+    operation DecodeMessageFromQubits(qAlice : Qubit, qBob : Qubit) : (Bool, Bool) {
+        // Implement your solution here...
+
+        // Returning the qubit states after measuring
+        return (MResetZ(qAlice) == One, MResetZ(qBob) == One);
+    }
+
+    // You might find this helper operation from an earlier task useful.
+    operation CreateEntangledPair(qAlice : Qubit, qBob : Qubit) : Unit is Adj {
```

Comment:
I thought we were going to drop this helper from this task and assume that the learner will just type in CNOT and H?

Reply:
I thought of keeping it for the convenience of the learner, in case they might  want to use it. I was more of favor of dropping it here, in case we drop the last End-To-End task, which requires us to copy-paste all the 3 solutions. 

@tcNickolas , Kindly let me know if you think, this should be dropped here. In that case, I was thinking to update the solution text to avoid mention of `Adjoint`  functor.

Reply:
I would drop it here (but keep it in the e2e task, since you're right that we need all 3 solutions in it anyways). We can mention the Adjoint in the solution text even if we're not using it in the code - it's a useful bit of knowledge that using adjoint of state prep allows us to do measurement in Bell state even if we choose to implement adjoint by hand

Reply:
@tcNickolas, done.